pkgs/top-level/python-packages.nix

```diff
@@ -2794,6 +2794,8 @@ in {
 
   imageio = callPackage ../development/python-modules/imageio { };
 
+  imageio-ffmpeg = disabledIf isPy27 (callPackage ../development/python-modules/imageio-ffmpeg { });
```

Comment:
```suggestion
  imageio-ffmpeg = callPackage ../development/python-modules/imageio-ffmpeg { };
```

Reply:
I don't understand. imageio-ffmpeg needs Python 3. Is it wrong to disable the package for 2.7 here?

Reply:
Yes, please use `disabled = !isPy3k` in the expression instead.